sympy/stats/rv.py

```diff
@@ -1040,6 +1042,7 @@ def where(condition, given_condition=None, **kwargs):
     return pspace(condition).where(condition, **kwargs)
 
 
+@doctest_depends_on(modules=('scipy',))
 def sample(expr, condition=None, size=(), library='scipy',
            numsamples=1, seed=None, **kwargs):
```

Comment:
Maybe everything after `size` should be keyword-only. We don't want people use positional parameters here especially if the ``numsamples`` parameter will be removed.

Reply:
I think we can handle that in the next PR. We should make size and numsamples keyword only for compatibility since they will get replaced by shape which can be set as positional.